api-server/server/utils/donation.js

```diff
@@ -160,10 +160,7 @@ export async function cancelDonation(body, app) {
 
     User.findOne({ where: { id: userId } }, (err, user) => {
       if (err || !user || !user.donationEmails) throw Error(err);
-      log('Updating user record for donation cancellation');
-      user.updateAttributes({
-        isDonating: false
-      });
+      log('Updating donation record for donation cancellation');
     });
```

Comment:
Can't we simply remove this update entirely?

Reply:
Yup - I do not see why make an expensive call here if the goal is not to set anything.

Reply:
That makes sense.